text/0550-macro-future-proofing.md

```diff
@@ -12,7 +12,7 @@ possible to lift these restrictions backwards compatibly if desired.
 
 # Key Terminology
 
-- `macro`: anything invokable as `foo!(...)` in source code.
+- `macro`: anything invocable as `foo!(...)` in source code.
```

Comment:
invocable is an alternate spelling of invokable so this is not a typo.

Reply:
Merriam-Webster does not list "invokable" as a word. While some sources do list it, Google ngrams shows "invocable" has been consistently used roughly twice as much over the past couple decades (6x for American English, which Rust supposedly prefers). Given these two facts, I think "invocable" is the choice we should make.